src/features/island/plots/Plot.tsx

```diff
@@ -67,6 +67,7 @@ export const Plot: React.FC<Props> = ({ id }) => {
     plotIndex: id,
     crops: crops,
     buildings: buildings,
+    bumpkin: bumpkin,
```

Comment:
Just a note that when assigning a value to a key with the same name you can just use the key. 
```suggestion
    bumpkin
```